<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTextBrowser" name="tb_laps"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pb_lap">
      <property name="text">
       <string>Lap</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pb_exit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbl_time">
      <property name="text">
       <string>00:00:00</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pb_startstop">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="pb_clear">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>385</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_clear</sender>
   <signal>clicked()</signal>
   <receiver>tb_laps</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
